README.md

```diff
@@ -1,18 +1,57 @@
 # xSLUG
+![build status](https://travis-ci.org/exercism/x-api.svg?branch=master)
```

Comment:
For the build status image? Yeah I was going to ask about that. I was trying to have it display a "real" status image. `xslug` gives `{"file":"not found"}` but we could use `x-template` which gives ![](https://api.travis-ci.org/exercism/x-template.svg?branch=master)